pkg/pillar/scripts/device-steps.sh

```diff
@@ -339,10 +341,17 @@ if [ -f /var/run/watchdog.pid ]; then
 fi
 /usr/sbin/watchdog -c $TMPDIR/watchdogclient.conf -F -s &
 
-if ! [ -f $CONFIGDIR/device.cert.pem ] || ! [ -f $CONFIGDIR/device.key.pem ]; then
+if [ ! -f $CONFIGDIR/device.cert.pem ]; then
     echo "$(date -Ins -u) Generating a device key pair and self-signed cert (using TPM/TEE if available)"
     touch $CONFIGDIR/self-register-pending
     sync
+    if [ -c $TPM_DEVICE_PATH ]; then
```

Comment:
I thought we wanted a file (in /persist/config or /config) which would disable the use of the TPM so that we can create that file when we want to run LISP demos.
This just makes that file exist when /dev/tpm0 exists, which isn't very useful.

Reply:
Addressed in the latest commit (/config/disable-tpm)